src/components/structures/ViewSource.tsx

```diff
@@ -1,7 +1,6 @@
 /*
-Copyright 2015, 2016 OpenMarket Ltd
 Copyright 2019 Michael Telatynski <7t3chguy@gmail.com>
-Copyright 2019 The Matrix.org Foundation C.I.C.
+Copyright 2015, 2019, 2023 The Matrix.org Foundation C.I.C.
```

Comment:
This doesn't look completely right

Reply:
Oops, missed 2016, but got that now. I tried to merge the OpenMarket and Matrix.org lines [per a recent-ish message on Synapse](https://github.com/matrix-org/synapse/issues/7565#issuecomment-637561222), but I'm not totally clear what the actual rules are.